core/math/math_funcs.cpp

```diff
@@ -28,6 +28,7 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+#include "core/error_macros.h"
```

Comment:
This should come after `"math_funcs.h"`, see http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

Reply:
Oops. I've updated the commit.